_pages/user-experience/sign-in-sign-out.md

```diff
@@ -20,4 +20,201 @@ sidenav:
+    </div>
+</div>
+</div>
+<div class="grid-row margin-top-2 padding-bottom-2">
+  <div class="maxw-mobile-lg">
+    <img src="{{ site.baseurl }}/assets/img/sign-in-menu.svg"
+        alt='An example agency website header with two buttons labeled "Menu options" and "Sign in" at top right"'
+        class="display-block grid-col-auto green-bottom-border" />
+    <div class="text-green float-left">
+      <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"/></svg>
+      </div>
+      <div class="margin-left-3 maxw-mobile-lg font-body-xs">
+      <span class="text-uppercase text-bold text-green">Do</span>: Use your own site’s global navigation styling if “Sign in” appears within a list of menu options in the header.
+    </div>
+  </div>
+</div>
+
+After a user signs in, a sign-out button labeled “Sign out” replaces the sign-in button on the website header.
+
+### Optional: Sign-in options on a page
+
+Some agency partners decide to implement a sign-in page on their application prior to redirecting their users to Login.gov, especially to present multiple sign-in options.
+
+<div class="maxw-mobile-lg">
+  <img src="{{ site.baseurl }}/assets/img/global-sign-in.png"
+       alt='An example agency website page with the heading "Sign in" and three full width buttons labeled "Sign in with Login.gov" and "Sign in with [alternate provider]"'
+       class="display-block grid-col-auto green-bottom-border" />
+  <div class="text-green float-left">
+    <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"/></svg>
+    </div>
+    <div class="margin-left-3 font-body-xs">
+    <span class="text-uppercase text-bold text-green">Do</span>: Design sign-in options that are equal in size. Stacked, solid-colored buttons work best because the stacking supports responsive layouts and mobile views.
+  </div>
+</div>
+
+{% capture do_button_styling %}
+Do use Login.gov’s preferred button style or primary button styling, since signing in is the main action and helps your users know that it is the next step in the process.
+{% endcapture %}
+{% capture do_sign_in_label %}
+Do use the label “Sign in with Login.gov” to differentiate from other sign-in options you may have on your application.
+{% endcapture %}
+{% capture do_equal_size %}
+Do design sign-in options that are uniform in width and height.  Consistency and uniform sizes helps users easily scan information.
+{% endcapture %}
+{% capture do_list_style %}
+Do style each button in a list with a different background color and center-justify the text in the buttons. 
```

Comment:
This is showing up with a red don't check in front of the sentence. 
<img width="457" alt="Screenshot 2024-01-16 at 12 48 58 PM" src="https://github.com/18F/identity-dev-docs/assets/90725258/78f9d7e7-bc4a-48f5-93f2-aa333511c3a0">